setup.py

```diff
@@ -26,20 +26,20 @@
     platforms=['Any'],
     packages=['scrapinghub', 'scrapinghub.client', 'scrapinghub.hubstorage'],
     package_data={'scrapinghub': ['VERSION']},
-    install_requires=['requests>=1.0', 'retrying>=1.3.3', 'six>=1.10.0'],
-    extras_require={'msgpack': mpack_required},
+    install_requires= mpack_required + [
```

Comment:
Why is msgpack no longer optional?

Reply:
I have just found out that removing the entry from `extras_require` is backward incompatible, so I suggest we keep it with an empty list of requirements, as suggested [in the setuptools documentation](https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#optional-dependencies):

> Best practice: if a project ends up not needing any other packages to support a feature, it should keep an empty requirements list for that feature in its extras_require argument, so that packages depending on that feature don’t break (due to an invalid feature name).

Reply:
https://github.com/scrapinghub/python-scrapinghub/pull/164

(I would usually provide a suggestion, but I needed a fork I could clone for immediate usage)